lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2736,6 +2750,47 @@ def _add_image_features(self, node_space: schema.NodeSpace) -> None:
         else:
             ...
 
+        if (
+            node_space.disk
```

Comment:
`node_space.disk` can be asserted. So the if logic is simpler.

Reply:
If asserted node_space.disk, most of the unit tests will fail because of node_space.disk assert error.